### PR TITLE
Save the products to an MongoDB database 

### PR DESCRIPTION
**Pull request by RaghadAbusamor**

### CODE REVIEW

File: InventoryManagementSystem_MongoDB/DataBaseOperations/InventoryService.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+            return allProducts.ToString();
+        }
+
+        public async Task EditProductNameAsync(Product product, string? newName)
+        {
+            await _repository.EditProductNameAsync(product.Name, newName);
+        }
+
+        public async Task EditProductPriceAsync(Product product, decimal newPrice)
```
Comment (mohammadmo116) [inline InventoryManagementSystem_MongoDB/DataBaseOperations/InventoryService.cs:46]: no need to pass the whole object if not used
the name can be sufficient

**Review by mohammadmo116 — APPROVED** [review]
nice work, nice Clean Code
شغل حلو للامانه
approved with one comment

File: InventoryManagementSystem_MongoDB/DataBaseOperations/InventoryService.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
+﻿using System.Text;
+
+namespace InventoryManagementSystem
+{
+    public class InventoryService
+    {
+        private IProduct _repository;
```
Comment (hayatzsala) [inline InventoryManagementSystem_MongoDB/DataBaseOperations/InventoryService.cs]: same comment as PR#4 about class/interface naming

**Review by hayatzsala — APPROVED** [review]
approved but same comments for shared files between this PR and PR#4 about interface/classes naming

**Pull request merged**